### PR TITLE
Fix indentation and python version in crs2-renumbering script

### PR DESCRIPTION
**Pull request by j0k2r**

### CODE REVIEW

**Comment by fgsch** [comment]
Can we keep python 2 for the time being, and import print_function from __future__ so we can switch to python 3 at a later date?

**Comment by csanders-git** [comment]
I see no reason why we couldn't do that @fgsch

**Comment by csanders-git** [comment]
@j0k2r  and @fgsch, i migrated back to Python2 as we will be removing this script by 2020 :). I also upgraded the styling in accordance with pep8

File: util/crs2-renumbering/update.py [hunk 1]
```diff
@@ -1,40 +1,56 @@
+"""This is designed to convert 2.x CRS ID numbering to 3.x CRS numbering"""
+
```
Comment (fgsch) [inline util/crs2-renumbering/update.py]: You still need to move this below :)

**Comment by csanders-git** [comment]
So it looks like I screwed this up and therefore it is up to me to fix it :)

**Comment by csanders-git** [comment]
Not sure what i did to screw this up but rebasing isn't helping -- gonna close and reopen this as a new PR.

**Pull request closed without merging**